test/lib/mayaUsd/fileio/testCacheToUsd.py

```diff
@@ -564,10 +564,8 @@ def runTestMayaRefPrimTransform(self, createMayaRefPrimFn, checkCacheParentFn):
         checkCacheParentFn(self, cacheParentChildren, variantSet, cacheVariantName)
 
         # Maya reference prim should now have the updated transformation.
-        editTarget = self.stage.GetEditTarget()
-        if variantSet:
-            variantSet.SetVariantSelection('Rig')
-            editTarget = variantSet.GetVariantEditTarget(editTarget.GetLayer())
+        if variantSetName:
+            cacheParent.GetVariantSet(variantSetName).SetVariantSelection('Rig')
```

Comment:
The code was doing unnecessary things. (I think this block was copy/paste from earlier in the same test.) I had to go through this unit test in early prototypes and fixed it back then, so I left the change in, even though it is not strictly necessary for this PR.